crates/iota/src/name_commands.rs

```diff
@@ -507,65 +558,86 @@ impl NameCommand {
                     .id();
                 let iota_names_config = get_iota_names_config(&iota_client).await?;
 
-                NameCommandResult::Client(
-                    IotaClientCommands::Call {
-                        package: IOTA_FRAMEWORK_PACKAGE_ID,
-                        module: "transfer".to_owned(),
-                        function: "public_transfer".to_owned(),
-                        type_args: vec![TypeTag::from_str(&format!(
-                            "{}::iota_names_registration::IotaNamesRegistration",
-                            iota_names_config.package_address
-                        ))?],
-                        args: vec![
-                            IotaJsonValue::from_object_id(nft_id),
-                            IotaJsonValue::new(serde_json::to_value(address)?)?,
-                        ],
-                        gas_price: None,
-                        opts,
-                    }
-                    .execute(context)
-                    .await?,
-                )
+                let res = IotaClientCommands::Call {
+                    package: IOTA_FRAMEWORK_PACKAGE_ID,
+                    module: "transfer".to_owned(),
+                    function: "public_transfer".to_owned(),
+                    type_args: vec![TypeTag::from_str(&format!(
+                        "{}::iota_names_registration::IotaNamesRegistration",
+                        iota_names_config.package_address
+                    ))?],
+                    args: vec![
+                        IotaJsonValue::from_object_id(nft_id),
+                        IotaJsonValue::new(serde_json::to_value(address)?)?,
+                    ],
+                    gas_price: None,
+                    opts,
+                }
+                .execute(context)
+                .await?;
+
+                handle_transaction_result(res, async |res| {
+                    Ok(NameCommandResult::Transfer {
```

Comment:
We are not showing more than the parameters here, I would revert to showing the usual output

Reply:
It's very verbose for no real reason IMO

Reply:
moot point now that we have the verbose flag